src/escape.rs

```diff
@@ -2181,7 +2181,7 @@ mod normalization {
                     if ch == '\u{0085}' {
                         assert_eq!(normalize_xml11_eols(input), "\n", "{}", description);
                     } else {
-                        let mut expected = utf8.clone();
+                        let mut expected = utf8;
```

Comment:
That clone is intentional. We do not want to change `utf8` content, because that will change the `input`. I even doubt if that could be compiled.